src/store/actions/words.js

```diff
@@ -0,0 +1,8 @@
+import { getWords } from '../../service/service';
+
+export const INIT_WORDS = 'INIT_WORDS';
+
+export const initWords = () => ({
```

Comment:
I think it's cleaner and more predictable if action name matches file name (so maybe -> `init-words`). It makes things easier to find…

Reply:
hm, I thought actions' file names should match reducers' ones (?)

Reply:
no, actions are actions, reducers are reducers

please keep in mind action-reducer is not 1:1 relationship
an action can be handled by multiple reducers
for example `resetSomething` may cause several reducers to update state with a default value